src/app/fyle/my-profile/my-profile.page.ts

```
@@ -546,14 +546,14 @@
   async updateMobileNumber(eou: ExtendedOrgUser): Promise<void> {
     this.trackingService.updateMobileNumberClicked({
-      popoverTitle: (eou.ou.mobile?.length ? 'Edit' : 'Add') + ' Mobile Number',
+      popoverTitle: (eou.ou.mobile?.length ? 'Edit' : 'Add') + ' mobile number',
     });
 
     const updateMobileNumberPopover = await this.popoverController.create({
       component: UpdateMobileNumberComponent,
       componentProps: {
-        title: (eou.ou.mobile?.length ? 'Edit' : 'Add') + ' Mobile Number',
-        inputLabel: 'Mobile Number',
+        title: (eou.ou.mobile?.length ? 'Edit' : 'Add') + ' mobile number',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)